vue/src/components/Results.vue

```diff
@@ -223,7 +223,30 @@ export default {
       }
     }
   },
-  props: ['input', 'single']
+  props: ['input', 'single'],
+  watch: {
+    input: {
+      handler(newInput, oldInput) {
+        if (newInput && (!oldInput || newInput.runId !== oldInput.runId)) {
```

Comment:
I wonder if there's an equivalent of React's <Results key={input.runId}/>, which will just create a fresh Results for each key.  This watch method works but is a bit painful to remember to do in every component.